puppet/zulip/manifests/rabbit.pp

```diff
@@ -46,17 +46,10 @@
 
   $rabbitmq_nodename = zulipconf('rabbitmq', 'nodename', '')
   if $rabbitmq_nodename != '' {
-    file { '/etc/rabbitmq':
-      ensure => 'directory',
-      owner  => 'root',
-      group  => 'root',
-      mode   => '0755',
-    }
-
     file { '/etc/rabbitmq/rabbitmq-env.conf':
       ensure  => file,
-      require => File['/etc/rabbitmq'],
-      before  => [Package[rabbitmq-server], Service[rabbitmq-server]],
+      require => Package[rabbitmq-server],
+      before  => Service[rabbitmq-server],
```

Comment:
I suppose the `before` should ensure that we restart rabbitmq after replacing its configuration file?

Reply:
I see, you fix that in the next commit.

Reply:
I can combine the two commits if you'd prefer.

Reply:
If you think that's clearer, we can, but I think either approach is reasonable.